packages/java/parser-jvm-plugin-model/src/main/java/dev/hilla/parser/plugins/model/ModelPlugin.java

```diff
@@ -53,20 +72,31 @@ private static boolean isValidationConstraintAnnotation(
                 .startsWith(VALIDATION_CONSTRAINTS_PACKAGE_NAME);
     }
 
+    private static boolean isIncludedAnnotation(
+            AnnotationInfoModel annotation) {
+        return INCLUDED_ANNOTATIONS.contains(annotation.getName());
+    }
+
     @Override
     public void enter(NodePath<?> nodePath) {
-        if (!(nodePath.getNode() instanceof TypedNode)) {
+        if (!(nodePath.getNode()instanceof TypedNode typedNode)) {
```

Comment:
```suggestion
        if (!(nodePath.getNode() instanceof TypedNode typedNode)) {
```

Reply:
Unfortunately that is how `mvn formatter:format` decides to format this.

Reply:
That's weird. Probably, they don't know what to do with that new syntax. But ok, let's have it that way then.

Reply:
Oh look it was resolved with the latest formatter :)